import-service/handlers/importFileParser.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+import AWS from 'aws-sdk';
+import { BUCKET, STATUS_CODES, AWS_REGION } from '../utils/constants';
+import lambdaWrapper from '../utils/lambdaWrapper';
+
+export const importFileParser = lambdaWrapper(async (event) => {
+  const s3 = new AWS.S3({ region: AWS_REGION });
```

Comment:
Take advantage of execution environment reuse to improve the performance of your function. Initialize SDK clients and database connections outside of the function handle. Subsequent invocations processed by the same instance of your function can reuse these resources. This saves cost by reducing function run time.

Read more here: https://docs.aws.amazon.com/lambda/latest/dg/best-practices.html